deeppavlov/dataset_readers/snips_intents_reader.py

```diff
@@ -28,8 +28,8 @@
 log = get_logger(__name__)
 
 
-@register('classification_datasetreader')
-class ClassificationDatasetReader(DatasetReader):
+@register('snips_intents_reader')
+class SnipsIntentsDatasetReader(DatasetReader):
```

Comment:
This class works with generic csv-files for classification, not just Snips dataset

Reply:
Ok, done.